packages/larva/lib/writeJson.js

```diff
@@ -18,27 +18,42 @@ module.exports = function writeJson( patternConfig, fromLarva = false ) {
 
 		const startPath = sourceDirectory + '/modules/' + moduleName;
 		const variants = getPatternVariants( startPath );
-		
+
 		variants.forEach( ( variant ) => {
 
-			moduleData = getPatternData( sourceDirectory, {
+			const moduleData = getPatternData( sourceDirectory, {
 				type: 'modules',
 				name: moduleName,
 				variant: variant
 			} );
 
-			jsonDestPath = path.resolve( patternConfig.projectPatternsDir, '../../build/json/modules/' + moduleName + '.' + variant + '.json' );
-
-				// If JSON data and module prototype are the same, pass, otherwise write the data to file
-				if ( fs.existsSync( jsonDestPath ) && JSON.stringify( require( jsonDestPath ) ) === JSON.stringify( moduleData ) ) {
-					console.log( chalk.grey( `No updates in ${moduleName}.${variant}` ) );
-				} else {
-					if ( undefined !== moduleData ) {
-						writeJsonToFile( jsonDestPath, moduleData );
-						console.log( chalk.green.bold( `Wrote JSON for ${moduleName}.${variant}` ) );
-					}
+			const jsonDestPath = path.resolve( patternConfig.projectPatternsDir, '../../build/json/modules/' + moduleName + '.' + variant + '.json' );
+
+			// If JSON data and module prototype are the same, pass, otherwise write the data to file
+			let hasChanged = true;
+			try {
+				if (
```

Comment:
Please remove new lines within the if condition - this is an opinionated code style change that makes the changes harder to see, and should be in a separate PR. 

This style will make the code in this file look different than other files in the project  - if there are advantages to structuring statements like this, we can add a `max-len` value to the eslint config and update the source files in a PR specific to that.

Reply:
updated